src/bench/pqs/linden.cpp

```diff
@@ -35,16 +36,38 @@ Linden::~Linden()
 
 void
 Linden::insert(const uint32_t &key,
-               const uint32_t & /* Unused */)
+               const uint32_t &value)
 {
-    linden_insert(m_q->pq, key);
+    /*Add 1 to key to support 0 keys*/
+    linden_insert(m_q->pq, key+1, value);
+}
+
+void
+Linden::insert(const size_t &key,
```

Comment:
I think it is an excellent idea to settle on 64-bit keys and values. I think 64-bit would be better than 32-bit as:

* that is what the priority queues that are implemented in C (Linden, SprayList, CA-PQ) are hard coded for on modern platforms and
* it would make it possible to fit a memory address in the value field.

However, I think such a change is not related to this pull request so perhaps it is better to do this as a separate commit once this pull request has been accepted. What do you think?